docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -16,7 +16,116 @@ MacOS setup features require connecting Fleet to Apple Business Manager (ABM). L
 
 ## End user authentication
 
-> This feature is currently in development.
+Using Fleet, you can require end users to authenticate with your identity provider (IdP) and agree to an end user license agreement (EULA) before they can use their new Mac.
+
+To require end user authentication, we will do the following steps:
+
+1. Connect Fleet to your IdP
+2. Upload a EULA to Fleet
+3. Enable end user authentication
+
+### Step 1: connect Fleet to your IdP
+
+In Fleet, you can connect Fleet to your IdP using the Fleet UI or fleetctl command-line tool.
+
+The Fleet UI method is a good start if you're just getting familiar with Fleet.
```

Comment:
I don't think we need to explain this here or the part below.

Reply:
Good call. Removed it.